### DISCUSSION TITLE
Homescreen for new projects now missing

### DISCUSSION THREAD

**Discussion by MildTomato** [supabase/supabase#16845] [Contribute to Supabase]
# Bug report

## Describe the bug

Made a new free project - I already have existing free projects.
The home screen is now the stats state, rather than the 'welcome screen' state.

## To Reproduce

see description above

## Expected behavior

Maybe show the welcome screen. Maybe this now hides welcome screen if it's the second project?

## Screenshots

My new project
<img width="1840" alt="Screenshot 2023-02-05 at 5 19 25 PM" src="https://user-images.githubusercontent.com/8291514/216811113-3294ccd3-e1bc-4fd0-bdb3-8a0f407291de.png">

## System information

- OS: macosx
- Browser (if applies) chrome latest

**Comment by Hallidayo** [comment]
Hi @MildTomato, @alaister and @joshenlim - I've been looking at this one and it seems when you create a new project it seems that some API requests get sent to `/rest-admin/v1/live` straight away.

Should the `NewProjectPanel` component only been show if their are tables and users created in the project?

**Comment by MildTomato** [comment]
I remember working on this (a long time ago) and I think I ended up checking some obscure number for database size, (since it's never ever 0kb) as there is a specific db size for a clean new project you can check, to determine wether to show/hide welcome messages - but maybe that got taken out and replaced with something else. 

@joshenlim assuming you might know?

**Comment by joshenlim** [comment]
hmm the logic is in `ProjectUsageSection`

```
const hasProjectData = usage
    ? Object.keys(usage)
        .map((key) => usage[key as keyof ProjectUsageResponse].usage)
        .some((usage) => (usage ?? 0) > 0)
    : false
```

So we're checking all the usage properties from the useProjectUsageQuery, if anything is above 0, show the stats
I'll need to investigate a little to see which metric has > 0 upon project creation

**Comment by Hallidayo** [comment]
@joshenlim - When I console out usage object on the default project it seems that both `bucketSize` and `dbSize` have values greater than 0.

`authUsers` and `dbTables` have values of 0.
